project/Dependencies.scala

```diff
@@ -0,0 +1,11 @@
+import sbt._
+
+object Dependencies {
+  lazy val scalaTest = "org.scalatest" %% "scalatest" % "3.1.0"
+  lazy val pluginsApi = "com.codacy" %% "codacy-plugins-api" % "4.0.1"
+
+  val circeVersion = "0.12.3"
```

Comment:
How safe is it to use multiple versions of circe?

Reply:
How is this using multiple versions of circe?

Reply:
I meant one here and another in the other side.

Reply:
As long as the serialization is done correctly there should be no problem. And it is done correctly